validator/build.py

```diff
@@ -300,5 +320,7 @@ def RunValidatorTest(out_dir):
 GenerateValidateBin(out_dir='codegen')
 RunSmokeTest(out_dir='codegen')
 CompileValidatorTestMinified(out_dir='codegen')
-GenerateValidatorTest(out_dir='codegen')
-RunValidatorTest(out_dir='codegen')
+CompileHtmlparserTestMinified(out_dir='codegen')
```

Comment:
Went with dist, since build conflicts with the Python module name build.py. In the longer run this will be a non-issue since hopefully we'll get rid of build.py and use whatever javascript stuff makes sense.